modules/oe_content_person/oe_content_person.post_update.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Person Content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add length restrictions for title display fields.
+ */
+function oe_content_person_post_update_20001(): void {
+  // Update form display to add maxlength to first and last names.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_person') . '/config/post_updates/20001_person_title');
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $display_values = $storage->read('core.entity_form_display.node.oe_person.default');
```

Comment:
We have to update not this config, but field.storage.

Reply:
It was decided together with business that for first and last name we don't update the storage, but rather only add the restriction in the form. If you see the ticket description it was updated to reflect this.

Reply:
I got it